doc/contributing/managing-social-media-acounts.md

```diff
@@ -0,0 +1,49 @@
+# Managing social media accounts
+
+The project obtains and manages social media accounts in order to help
+promote the project and to share content that benefits the ecosystem.
+
+Currently the project has the following social media accounts:
+
+* [bluesky](https://bsky.app/profile/nodejs.org)
+* [twitter](https://x.com/nodejs)
+* [Facebook](https://www.facebook.com/nodejsfoundation)
+* [LinkedIn](https://www.linkedin.com/company/node.js-foundation/)
+
+## bluesky
+
+Bluesky is managed directly by the project. Automation is used
+to request posts/reposts as outlined in
+[nodejs/bluesky](https://github.com/nodejs/bluesky).
+
+Login credentials are available to members of the Technical
+Steering Committee, but are not to be used directly except
+in exceptional circumstances.
+
+## Twitter/X
+
+Day to day management of the Twitter/X account has been
+delegated to the Foundation Staff.
+
+In addition, it has been agreed that Releasers may login
+and post when doing a release, and TSC members may post
+an announcement of a release if a releaser is not able to
+do it themselves.
+
+Login credentials are available to members of the Technical
+Steering Committee. Releasers may request access by
+opening an issue in the nodejs/admin repo asking for access.
+
+## Facebook
+
+Day to day management of the Facebook account has been
+delegated to the Foundation Staff.
+
+No project members have access to the Facebook account.
+
+## Linkedin
```

Comment:
Considering Linkedin is also part of the release process (to announce whenever we release some important blog posts or new versions), I think it might make sense to have it as well. 

For instance, I have asked for repost/retweet in https://openjs-foundation.slack.com/archives/C0142A39BNE/p1741372572005219 and no action so far.

Reply:
Consider this as non-blocking.